eval/chat_benchmarks/AIME24/eval_instruct.py

```
@@ -0,0 +1,214 @@
+            self.logger.info(f"Processing {len(examples)} examples")
+
+            all_instances = []
+            for idx, example in enumerate(examples):
+                try:
+                    inputs = model.apply_chat_template([{"role": "user", "content": PROMPT.format(problem=example['problem'])}])
+                    
+                    all_instances.append(
+                        Instance(
+                            "generate_until",
+                            example,
+                            (
+                                inputs,
+                                {
+                                    "max_gen_toks": self.max_tokens,
+                                    "do_sample": False,
+                                },
+                            ),
+                            idx,
+                        )
+                    )
+                except Exception as e:
+                    self.logger.error(f"Error preparing instance {idx}: {str(e)}")
+                    continue
+
+            self.logger.info("Generating responses for AIME24...")
+            outputs = self.compute(model, all_instances)
+
+            # Return None early for non-primary ranks
+            if model.rank != 0:
+                return None
+
+            generated_examples = []
+            for example, output in zip(examples, outputs):
+                try:
+                    example_with_output = example.copy()
+                    example_with_output["output"] = output
+                    processed_output = extract_answer(output)
+                    example_with_output["processed_output"] = processed_output
+                    print(f'example: {example}')
+                    print(f'output: {output}')
+                    print(f'processed output: {processed_output}')
+                    generated_examples.append(example_with_output)
+                except Exception as e:
```

If possible try to catch a specific exception rather than broad catch-all exception.

---

actually, we don't need this block since the data file is already added into the repo. i had copied from one of the other eval_instruct files in the codebase. i will remove the try-catch block now.